src/docfx/build/metadata/MetadataProvider.cs

```diff
@@ -87,7 +87,10 @@ public MetadataProvider(Docset docset, Cache cache, MicrosoftGraphCache microsof
                 }
             }
 
-            errors.AddRange(_schemaValidator.Validate(result));
+            if (file.ContentType == ContentType.Page && string.IsNullOrEmpty(file.Mime))
```

Comment:
>file [](start = 16, length = 4)

If (file.ContentType != ContentType.TableOfContents)

Reply:
There is now a `Document.IsConceptual` property

Reply:
Seems he only wants to validate markdown files.

Reply:
Sorry for confuse, I have added a commend.